src/librustc/diagnostics.rs

```diff
@@ -1587,6 +1587,52 @@ This will fail because the compiler does not know which instance of `Foo` to
 call `bar` on. Change `Foo::bar()` to `Foo::<T>::bar()` to resolve the error.
 "##,
 
+E0283: r##"
```

Comment:
You need to remove the error code from that other error list that lists all the error codes without explanations.